releasetasks/templates/firefox/enUS.yml.tmpl

```diff
@@ -157,12 +157,14 @@
                 {% if signing_class == "dep-signing" %}
                 MOZ_DISABLE_MAR_CERT_VERIFICATION: true
                 SIGNING_CERT: "dep"
-                {% else %}
+                {% elif signing_class == "nightly-signing" %}
+                SIGNING_CERT: "nightly"
+                {% elif signing_class == "release-signing" %}
                 SIGNING_CERT: "release"
                 {% endif %}
-                {% if branch == "mozilla-beta" %}
+                {% if accepted_mar_channel_id %}
```

Comment:
Assuming that this variable is always defined. Otherwise you would need to use something like
```
{% if accepted_mar_channel_id is defined and accepted_mar_channel_id %}
```